bundle/config/validate/all_resources_defined.go

```diff
@@ -0,0 +1,47 @@
+package validate
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/dyn"
+)
+
+func AllResourcesHaveValues() bundle.Mutator {
+	return &allResourcesHaveValues{}
+}
+
+type allResourcesHaveValues struct{}
+
+func (m *allResourcesHaveValues) Name() string {
+	return "validate:all_resources_have_values"
+}
+
+func (m *allResourcesHaveValues) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
+	rv := b.Config.Value().Get("resources")
+
+	// Skip if there are no resources block defined, or the resources block is empty.
```

Comment:
Isn't it an error or at least warning case? In which case we might not have resources block?

Reply:
Not having a `resources` block is allowed. Having a warning for an empty `resources` block makes sense to me. Should we make it more generic though? Like we emit a warning for any fields in the YAML configuration that only have the key specified but not the value.